js/main.js

```diff
@@ -1,26 +1,112 @@
-function getRandomNumber(min, max) {
-  let getQuantity = Math.floor(Math.random() * (max - min + 1) + min);
-  if (min >= max) {
-    getQuantity = (Math.random() * (max - min + 1) + min);
-  }
-  return getQuantity;
+function getRandomInteger(min, max) {
+  const lower = Math.min(Math.abs(min), Math.abs(max));
+  const upper =  Math.max(Math.abs(min), Math.abs(max));
+  const randomNumber = (Math.random() * (upper - lower) + lower);
+  return Math.floor(randomNumber);
 }
 
+getRandomInteger(20, 50);
+
 
-for (let i = 1; i <= 10; i++) {
-  getRandomNumber(0, 50);
+function getRandomNumberFloatingPoint(minNumber, maxNumber, floatingPoint) {
+  const lower = Math.min(Math.abs(minNumber), Math.abs(maxNumber));
+  const upper = Math.max(Math.abs(minNumber), Math.abs(maxNumber));
+  const  randomNumberFloat = (Math.random() * (upper - lower) + lower);
+  return randomNumberFloat.toFixed(floatingPoint);
 }
 
+getRandomNumberFloatingPoint(140, 100, 1);
+
+
+const typeHousing = ['palace', 'flat', 'house', 'bungalow', 'hotel'];
+const timeCheckin = ['12:00', '13:00', '14:00'];
+const timeCheckout = ['12:00', '13:00', '14:00'];
+const featuresHousing = ['wifi', 'dishwasher', 'parking', 'washer', 'elevator', 'conditioner'];
+const photos = ['https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg', 'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg', 'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg.'];
 
-function getRandomNumberFloatingPoint(minNumber, maxNumber, numSimbolsAfterCom) {
-  let getQuantitySecond = (Math.random() * (maxNumber - minNumber + 1) + minNumber);
-  if (minNumber >= maxNumber) {
-    getQuantitySecond = Math.floor(Math.random() * (maxNumber - minNumber + 1) + minNumber);
+const getArrayLength = () => {
```

Comment:
`getArrayLegth` means that you return length of array, that is number, but you return random length array of houseFeatures, so change the name. Also use houseTypes as parameter, so function will more universal and you will be able to create array of everything you want